lib/flash_flow/issue_tracker/pivotal.rb

```diff
@@ -74,13 +91,20 @@ def finish(story_id)
       def comment(story_id)
         story = get_story(story_id)
         if story
-          note_prefix = 'Shipped to production on'
           unless already_has_comment?(story, Regexp.new("^#{note_prefix}"))
-            story.notes.create(:text => Time.now.strftime("#{note_prefix} %m/%d/%Y at %H:%M"))
+            story.notes.create(:text => Time.now.strftime(note_time_format))
           end
         end
       end
 
+      def note_prefix
+        'Shipped to production on'
+      end
+
+      def note_time_format
+        "#{note_prefix} %m/%d/%Y at %H:%M"
```

Comment:
could we convert this to local time here in Los Angeles? 

Currently, it shows up in UTC, which means the comment says "Shipped to production 7 or 8 hours from now".

It's confusing because Tracker shows times in Pacific Time.

If that isn't a good option, then maybe put the time zone into the output.